### DISCUSSION TITLE
Releases page should also show pagination at the top

### DISCUSSION THREAD

**Discussion by robjbrain** [community/community#31493] [Other Feature Feedback, Questions, & Ideas]
It is common to want to browse releases starting a couple of pages back and finishing at the most recent.

e.g. my Laravel version is 9.18.0, that release is currently on page 3. In order to see what had changed I found the 9.18.0 release on page 3 and then scrolled up, to get to page 2 I think had to go to the very bottom of the page and click page 2 where as it would been convenient to also show the pagination at the top as well as the bottom.

It would make sense to include this pagination
<img width="778" alt="image" src="https://user-images.githubusercontent.com/10143910/187697649-968b0c9d-09ae-461e-8293-f413893da8fe.png">

At the top here in between the releases/tags and "find a release"

<img width="1160" alt="image" src="https://user-images.githubusercontent.com/10143910/187697960-558cfc3b-3d09-4425-8025-cb6b8d445d49.png">

**Comment by CaitlinSweeney** [comment]
I would also appreciate this feature. Its really tedious to search for tags and releases. You can't manually enter it the query either. Unless you know what the last tag in the table.

**Comment by muizzhaque** [comment]
It can indeed be more user-friendly to have pagination controls at both the top and bottom of the page. This way, users wouldn’t need to scroll all the way down to navigate to a different page, especially when the list of releases is long.

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @robjbrain! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
